package.json

```diff
@@ -182,5 +182,13 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/newrelic/node-newrelic.git"
-  }
+  },
+  "files": [
+    "lib/",
+    "api.js",
+    "index.js",
+    "stub_api.js",
+    "newrelic.js",
+    "bin/tracetractor"
```

Comment:
Unsure about this one as it looks to be a utility clients can leverage (based on the comment [at the top of the file](https://github.com/newrelic/node-newrelic/blob/master/bin/tracetractor#L4)).

If that is the case might it be worth separating all bin contents related to testing such as:

- `ca-gen.js`
- `cassandra-setup.sh`
- `clean.sh`
- `compare-bench-results.js`
- `docker-env-vars.sh`
- `docker-services.sh`
- `publish-docs.sh`
- `run-bench.js`
- `run-versioned-tests.sh`
- `smoke.sh`
- `ssl.sh`
- `test-naming-rules.js`
- `travis-install-gcc5.sh`
- `travis-install-mongo.sh`
- `travis-node.sh`
- `travis-setup.sh`
- `update-ca-bundle.sh`
- `update-cats.sh`

...Into their own bin folder within the `test/` directory, and all scripts/binaries intended for customer use such as:

- `tracetractor`
- maybe others...

...can remain in the top level bin?

After which the entry in this list can simply be changed to `bin/`.

Reply:
Great suggestion!  I like the idea of separating testing related bin files.